src/pages-and-resources/discussions/messages.js

```diff
@@ -2,20 +2,20 @@ import { defineMessages } from '@edx/frontend-platform/i18n';
 
 const messages = defineMessages({
   heading: {
-    id: 'authoring.forumSelectorTool.heading',
+    id: 'authoring.discussions.heading',
     defaultMessage: 'Which discussion tool would you like to use for this course?',
```

Comment:
i18n?

Reply:
These `messages.js` files are the actual English strings that get uploaded to Transifex and translated. :grin: So yeah!

Reply:
OMFG i did not notice the name of the file 🤦‍♀️ 🤦‍♀️ 🤦‍♀️ 🤦‍♀️ 🤦‍♀️